scripts/staging/bayesian_optimisation/test/bayesianOptimisationSinTest.dml

```diff
@@ -0,0 +1,47 @@
+# Test
+source("./scripts/staging/bayesian_optimisation/bayesianOptimisation.dml") as bayOpt;
+
+kernel_gaus = function(Matrix[Double] X1, Matrix[Double] X2, Double variance)
+  return (Matrix[Double] result)
+{
+    #TODO: What to do about multiple dimensional matrix?
```

Comment:
Hi,
Just quick feedback, in DML we use two-space indentation.

Reply:
Hi and thanks for the Feedback, I applied it in the last commit.